apps/charterafrica/src/assets/icons/UserIcon.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="auto" height="auto" viewBox="0 0 60 61" fill="none">
```

Comment:
Lets keep original names from Figma icons (to spot duplicates, etc.). I expect this file to be named `Type=user, Size=32, Color=Black.svg` or `Type=user, Size=32, Color=CurrentColor.svg` if we make the it use `currentColor`.